contracts/jobs/JobsManager.sol

```diff
@@ -488,9 +488,9 @@ contract JobsManager is ManagerProxyTarget, IVerifiable, IJobsManager {
             // Protocol slashes transcoder for failing verification (no finder)
             bondingManager().slashTranscoder(transcoder, address(0), failedVerificationSlashAmount, 0);
 
-            PassedVerification(transcoder, _jobId, _claimId, _segmentNumber);
```

Comment:
This was a bug - previously `receiveVerification` would fire the `PassedVerification` event when verification fails...This change makes sure `PassedVerification` is fired when verification succeeds.